src/main/scala/uk/gov/hmrc/mongo/ReactiveRepository.scala

```diff
@@ -20,8 +19,9 @@ abstract class ReactiveRepository[A <: Any, ID <: Any](collectionName: String,
                                                        idFormat: Format[ID] = ReactiveMongoFormats.objectIdFormats,
                                                        mc: Option[JSONCollection] = None)
                                                       (implicit manifest: Manifest[A], mid: Manifest[ID])
-  extends Repository[A, ID] with Indexes with ImplicitBSONHandlers {
+  extends Repository[A, ID] with Indexes {
```

Comment:
We don't want to mix in a sealed trait. This means we need explicitly import this in micro-services. A necessary evil for the greater good of being able to deprecate our reactivemongo in the future.